simplified-viewer-audiobook/src/main/java/org/nypl/simplified/viewer/audiobook/AudioBookPlayerActivity.kt

```diff
@@ -412,7 +412,7 @@ class AudioBookPlayerActivity : AppCompatActivity(),
         this.supportFragmentManager
           .beginTransaction()
           .replace(R.id.audio_book_player_fragment_holder, this.playerFragment, "PLAYER")
-          .commit()
+          .commitAllowingStateLoss()
```

Comment:
We didn't need that state anyway. :laughing:

Reply:
haha, yeah, I love that method. does the same thing but makes the fragment manager not complain so much. Another one of the dumb things about the Fragment API.